src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -5592,8 +5592,8 @@ def test_network_vpn_gateway_package_capture(self, resource_group, storage_accou
         self.cmd('network vnet create -g {rg} -n {vnet1} --subnet-name GatewaySubnet --address-prefix 10.0.0.0/16 --subnet-prefix 10.0.0.0/24')
         self.cmd('network vnet-gateway create -g {rg} -n {gw1} --vnet {vnet1} --public-ip-address {ip1} --sku {gw1_sku}')
         output = self.cmd('network vnet-gateway packet-capture start -g {rg} -n {gw1}').output.strip()
-        self.assertTrue('Successful' in output, 'Expected Successful in output.\nActual: {}'.format(output))
-        # currently we cannot create traffic by cli command. So it will return an error when stop.
+        self.assertTrue('Successful' in output, 'Expected Successful in output.\nActual: {}'.format(str(output)))
+         # currently we cannot create traffic by cli command. So it will return an error when stop.
```

Comment:
```suggestion
        self.assertTrue('Successful' in output, 'Expected Successful in output.\nActual: {}'.format(output))
        # currently we cannot create traffic by cli command. So it will return an error when stop.
```